### PR TITLE
SRCH-2574 update CircleCi for Ruby 2.7

### PR DESCRIPTION
**Pull request by MothOnMars**
## Summary
- update CircleCi orb & Ruby image
- add more Rubies to the CircleCi matrix
- use 2.7.5 in development
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review:
 
#### Functionality Checks
 
- [x] Code is functional, as tested locally.

- [x] Tests have been added or updated to cover the proposed changes. If not, please explain why tests aren't needed:
 
- [x] Automated checks pass, if applicable. If CodeClimate checks do not pass, explain reason for failures:
 
- [x] If your changes will be tested manually, you have run `bundle update` and committed your changes to Gemfile.lock. - N/A

- [x] You have merged the latest changes from the target branch (usually `master` or `main`) into your branch.
 
- [x] Your target branch is `master`, `main`, or `production`, or you have specified the reason for an alternate branch here:
 
- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [x] You have squashed your commits into a single commit (exceptions: your PR includes commits with formatting-only changes, such as required by Rubocop or Cookstyle, or if this is a feature branch that includes multiple commits)
 
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket
 
#### Process Checks

- [x] You have specified an "Assignee", and if necessary, additional reviewers

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -73,6 +76,10 @@ workflows:
             parameters:
               ruby_version:
                 - 2.6.6
+                - 2.7.1
+                - 2.7.2
                 - 2.7.3
+                - 2.7.4
+                - 2.7.5
```
Comment (MothOnMars) [inline .circleci/config.yml:83]: While it is still not clear what the highest available version will be from Brightbox, I figured we may as well test against 'em all. It's probably overkill, but it doesn't slow anything down, and will provide a high degree of confidence when we're ready to switch versions in production. We can remove the unneeded 2.7 versions when we remove 2.6.6.

**Review by MyNameIsMissing — APPROVED** [review]
LGTM

**Comment by MothOnMars** [comment]
@MyNameIsMissing , this needs a quick re-review. I pushed an additional commit to remove the unneeded Ruby versions from the test matrix: https://github.com/GSA/asis/pull/113/commits/98ea680b7d4b4293e9f62b6e3e7f4a9ebb3ff2b5

**Review by MyNameIsMissing — APPROVED** [review]
LGTM

**Pull request merged**